generator/integration_tests/CMakeLists.txt

```diff
@@ -44,12 +51,102 @@ target_link_libraries(
            google-cloud-cpp::iam_v1_policy_protos
            google-cloud-cpp::longrunning_operations_protos
            google-cloud-cpp::rpc_status_protos)
-set_target_properties(google_cloud_cpp_generator_testing_grpc_lib
+set_target_properties(google_cloud_cpp_generator_golden_protos
                       PROPERTIES CXX_CLANG_TIDY "")
 
-# Only define the tests if testing is enabled. Package maintainers may not want
-# to build all the tests everytime they create a new package or when the package
-# is installed from source.
-if (BUILD_TESTING)
-    add_subdirectory(golden)
-endif (BUILD_TESTING)
+file(
+    GLOB google_cloud_cpp_generator_golden_files
+    RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
+    "golden/*.h" "golden/*.cc" "golden/internal/*.h" "golden/internal/*.cc"
+    "golden/mocks/*.h")
```

Comment:
same comment about splitting out the mocks. it's fine if you don't think this is worth doing.

Reply:
Ack